UnitsNet/QuantityInfoLookup.cs

```diff
@@ -23,7 +23,7 @@ namespace UnitsNet;
 /// </remarks>
 internal class QuantityInfoLookup
 {
-    private readonly QuantityInfo[] _quantities;
+    private readonly IReadOnlyList<QuantityInfo> _quantities;
```

Comment:
there is a slight performance overhead of using the interface on the private field, but if you prefer clarity- it's fine by me..

Reply:
This change was collateral damage, I'll revert it. The idea was to accept IReadOnlyList as input and reuse instance if compatible, but I'll do the same for array instead.